src/main/java/com/mraof/minestuck/world/gen/structure/FrogTemplePiece.java

```diff
@@ -27,12 +26,16 @@
 import net.minecraft.world.level.levelgen.structure.pieces.StructurePieceSerializationContext;
 import net.minecraft.world.level.material.Fluids;
 
+import java.util.List;
+import java.util.Optional;
 import java.util.Random;
 
 public class FrogTemplePiece extends CoreCompatibleScatteredStructurePiece
 {
 	private boolean createRan = false; //boolean check to prevent certain objects(the lotus flower entity) from spawning several times over
 	private static final FrogTemplePiece.Selector HIEROGLYPHS = new FrogTemplePiece.Selector();
+	private static final List<WeightedEntry.Wrapper<Block>> weightedBlockList = buildWeightedList();
```

Comment:
Problem: You're calling `get()` on registry objects during static initialization, this is not good! You could either make the list non-static (so that it is initialized when the structure piece is created, which will not happen until after things have been loaded), or you could store them as suppliers instead of blocks.

Reply:
Made non static